electrumx/lib/coins.py

```diff
@@ -3842,3 +3842,198 @@ class Quebecoin(AuxPowMixin, Coin):
+    DESERIALIZER = lib_tx.DeserializerSegWit
+    BASIC_HEADER_SIZE = 112
+    TX_COUNT = 1
+    TX_COUNT_HEIGHT = 1
+    TX_PER_BLOCK = 1
+    RPC_PORT = 9245
+    REORG_LIMIT = 5000
+
+    @classmethod
+    def genesis_block(cls, block):
+        header = cls.block_header(block, 0)
+        header_hex_hash = hash_to_hex_str(cls.header_hash(header))
+        if header_hex_hash != cls.GENESIS_HASH:
+            raise CoinError('genesis block has hash {} expected {}'
+                            .format(header_hex_hash, cls.GENESIS_HASH))
+
+        return block
+
+    @classmethod
+    def electrum_header(cls, header, height):
+        version, = struct.unpack('<I', header[:4])
+        timestamp, bits, nonce = struct.unpack('<III', header[100:112])
+
+        return {
+            'block_height': height,
+            'version': version,
+            'prev_block_hash': hash_to_hex_str(header[4:36]),
+            'merkle_root': hash_to_hex_str(header[36:68]),
+            'timestamp': timestamp,
+            'bits': bits,
+            'nonce': nonce,
+        }
+
+
+class Potcoin(Coin):
+    NAME = "Potcoin"
+    SHORTNAME = "POT"
+    NET = "mainnet"
+    P2PKH_VERBYTE = bytes.fromhex("37")
+    P2SH_VERBYTES = [bytes.fromhex("05")]
+    WIF_BYTE = bytes.fromhex("b7")
+    GENESIS_HASH = ('de36b0cb2a9c7d1d7ac0174d0a89918f'
+                    '874fabcf5f9741dd52cd6d04ee1335ec')
+    DESERIALIZER = lib_tx.DeserializerPotcoin
+    TX_COUNT = 1
+    TX_COUNT_HEIGHT = 1
+    TX_PER_BLOCK = 1
+    RPC_PORT = 42000
+    REORG_LIMIT = 5000
+
+
+class Verge(Coin):
+    NAME = "Verge"
+    SHORTNAME = "XVG"
+    NET = "mainnet"
+    P2PKH_VERBYTE = bytes.fromhex("1E")
+    P2SH_VERBYTES = [bytes.fromhex("21")]
+    WIF_BYTE = bytes.fromhex("9E")
+    GENESIS_HASH = ('00000fc63692467faeb20cdb3b53200d'
+                    'c601d75bdfa1001463304cc790d77278')
+    DESERIALIZER = lib_tx.DeserializerTxTime
+    DAEMON = daemon.LegacyRPCDaemon
+
+    TX_COUNT = 1788355
+    TX_COUNT_HEIGHT = 1253018
+    TX_PER_BLOCK = 2
+    RPC_PORT = 20102
+    REORG_LIMIT = 5000
+    HEADER_HASH = None
+
+    @classmethod
+    def header_hash(cls, header):
+        if cls.HEADER_HASH is None:
+            import scrypt
+            cls.HEADER_HASH = lambda x: scrypt.hash(x, x, 1024, 1, 1, 32)
+        return cls.HEADER_HASH(header)
+
+
+class Vivo(Coin):
+    NAME = "Vivo"
+    SHORTNAME = "VIVO"
+    NET = "mainnet"
+    P2PKH_VERBYTE = bytes.fromhex("46")
+    P2SH_VERBYTES = [bytes.fromhex("0a")]
+    WIF_BYTE = bytes.fromhex("c6")
+    GENESIS_HASH = ('00000f6be3e151f9082a2b82c2916192'
+                    'a791090015b80979934a45d625460d62')
+    TX_COUNT = 1
+    TX_COUNT_HEIGHT = 1
+    TX_PER_BLOCK = 1
+    RPC_PORT = 9323
+    SESSIONCLS = DashElectrumX
+    DAEMON = daemon.DashDaemon
+    REORG_LIMIT = 8000
+
+    @classmethod
+    def header_hash(cls, header):
+        import neoscrypt
+        return neoscrypt.getPoWHash(header)
```

Comment:
I'm afraid we aren't able to take on neoscrypt coins until sparkspay/python-neoscrypt#1 has been resolved.